app/models/provider.rb

```diff
@@ -0,0 +1,11 @@
+class Provider
+  include ActiveModel::Model
+
+  attr_accessor :slug, :name
+
+  def self.all
+    I18n.t('.providers').map do |slug, name|
```

Comment:
```suggestion
    I18n.t('providers').map do |slug, name|
```

sans le point, ça n'a pas de sens dans un model les chemins relatifs.